core/controllers/base_test.py

```diff
@@ -269,6 +271,31 @@ def test_i18n_keys(self):
                     self.log_line('- %s' % key)
                 self.log_line('')
 
+    def alphabetic_test_i18n_keys(self):
+        """Tests that all JSON files are arranged in aplhabetical order."""
+        filenames = os.listdir(
+            os.path.join(os.getcwd(), self.get_static_asset_filepath(),
+                         'assets', 'i18n'))
+        for filename in filenames:
+            json_file_content = json.loads(utils.get_file_contents(
```

Comment:
No, because it uses json.loads(), which you want to avoid in this case. Instead, you want to analyze each line of the file as a text string, and use standard string operations to extract the key.